Userland/Services/AudioServer/ClientConnection.h

```diff
@@ -49,7 +49,7 @@ class ClientConnection final : public IPC::ClientConnection<AudioClientEndpoint,
     virtual Messages::AudioServer::GetPlayingBufferResponse get_playing_buffer() override;
     virtual Messages::AudioServer::GetMutedResponse get_muted() override;
     virtual void set_muted(bool) override;
-    virtual void set_sample_rate(u16 sample_rate) override;
+    virtual void set_sample_rate(u32 sample_rate) override;
```

Comment:
...you then changed it back to a `u32` here. Probably a good idea to remove them. :^)

Reply:
Yeah, my bad. Thanks for noticing it (:  I think I didn't see it because it ended up as u32, like if nothing had changed